src/opus.py

```diff
@@ -1,9 +1,24 @@
-from dataclasses import dataclass
-from typing import Dict
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional
 from pathlib import Path
 import aiofiles
 import yaml
 
+@dataclass
+class Asset:
+    """An asset describes a resource file"""
+    path: str
+
+
+@dataclass
+class ActionTemplate:
+    """An Action is a command triggered by a node"""
+    id: int
+    target: str
+    cmd: str
+    assets: List[str] = field(default_factory=lambda: [])
```

Comment:
field(default_factory=list) would also work here (same below, with dict). That's a matter of opinion though, so disregard if you prefer the lambda.